style.css

```diff
@@ -0,0 +1,161 @@
+* {
+margin: 0;
+padding: 0;
+box-sizing: border-box;
+}
+body {
+    font-family: 'Roboto' , sans- serif;
+}
+.header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 15px;
+}
+main {
+    height: calc(100vh - 70px);
+    display: flex;
+    background-color: #f9f9f9;
+}
+side-bar {
+    height: 100%;
+    width: 17%;
+    background-color: white;
```

Comment:
For side bars try using CSS Flexbox and CSS Grill, so that you can rearrange their position and size. As it is your first effort you have done well but there is room for improvement. You can refer to W3school CSS for more information about use of CSS flex and CSS grill.